packages/core/src/routes/connector.ts

```diff
@@ -122,7 +122,7 @@ export default function connectorRoutes<T extends AuthedRouter>(router: T) {
         await removeUnavailableSocialConnectorTargets();
       }
 
-      ctx.body = { ...connector, metadata, type };
+      ctx.body = transpileLogtoConnector({ dbEntry: connector, metadata, type });
```

Comment:
Is this relevant to the current PR?

Reply:
not that much, but I was trying to align the output of connector CRUD APIs.

Reply:
Better remove those changes, focus on current goal